besu/src/test/java/org/hyperledger/besu/PrivacyReorgTest.java

```diff
@@ -80,7 +80,6 @@
 import org.junit.rules.TemporaryFolder;
 
 @SuppressWarnings("rawtypes")
-// todo request lucas look at this pr
```

Comment:
did you get some feedback ?

Reply:
I did on that iteration. I'll ping him again for this one. :+1: